docs/concepts/standard-library/gnopher-hole.md

```diff
@@ -0,0 +1,105 @@
+---
+id: gnopher-hole-stdlib
```

Comment:
Still don't understand what is a "gnopher hole" and why this is labeled as such

Reply:
In order to appeal to both Golang devs & Web3 Devs, the idea is to have "learn more" or "go down the Gno rabbit hole" sections. The content in the Gnopher Hole is relatively complicated and not directly related to using Gno, but rather related with how to improve Gno itself - I see it as a section for people who want to go into the barebones implementation and dig deep on how Gno works.